release/llm_tests/serve/run_llm_serve_release_tests.py

```
@@ -42,9 +54,22 @@ def main(
     applications = get_applications(serve_config_file)
     compute_config = get_current_compute_config_name()
     env_vars = get_hf_token_env_var() if not skip_hf_token else {}
+    env_vars["VLLM_USE_V1"] = VLLM_USE_V1
+    llm_config = get_llm_config(serve_config_file)
 
+    if run_perf_profiler:
+        if image_uri is None:
+            cluster_env = os.environ.get("ANYSCALE_JOB_CLUSTER_ENV_NAME", None)
+            if cluster_env is not None:
```

should we raise an error if cluster_env is None? in that case image_uri will be none

---

Makes sense to me. Will also change this in the `start_service` logic.

(To be clear) seems like it's okay for image_uri to be None (am able to start a service from my workspace without this variable set).

But it makes sense to couple this tightly to the release test build environment to catch potential configuration drift.

---

You can do it in a workspace because on workspace we inherit the current dependency graph and ship it as the image_uri basically. It won't work when launching a job from arbitrary envs.